projectRouter.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+
+const router = express.Router();
+
+const projectDb = require('./data/helpers/projectModel');
+
+//GETS ALL PROJECTS
+router.get('/', (req, res) => {
+    projectDb.get()
+    .then(actions => {
+        res.status(200).json(actions)
+    })
+    .catch(err => {
+        res.status(500).json({ error: "GET FAILED!"})
+    })
+});
+
+
+//GETS ALL ACTIONS FOR A PROJECT
+router.get('/:id', validateProject, (req, res) => {
+   
+    projectDb.getProjectActions(req.project)
+    .then(action => {
+        res.status(200).json(action)
+    })
+    .catch(err => {
+        res.status(500).json({ error: "GET PROJECT ACTION FAILED!"})
+    })
+});
+
+
+//DELETES A PROJECT
+router.delete('/:id/actions', validateProject, (req, res) => {
+
+    projectDb.remove(req.project)
+    .then(actions => {
+        res.status(200).json(actions);
+    })
+    .catch(err => {
+        res.status(500).json({ error: "DELETE FAILED!"})
+   
+    })
+
+});
```

Comment:
**From our 1:1 -**

This would be a `.get` instead of a `.delete`, and it would return all of the actions that belong to one project.

We'd then have a separate 
```
router.delete("/:id" ...codeHere )
```
that actually deletes a project 👍